modules/@apostrophecms/i18n/i18n/en.json

```diff
@@ -171,6 +171,12 @@
   "manageDocType": "Manage {{ type }}",
   "manageDraftSubmissions": "Manage Draft Submissions",
   "managePages": "Manage Pages",
+  "mediaCreatedDateDayjsFormat": "MMM Do, YYYY",
+  "mediaCreatedDate": "Uploaded: {{ createdDate }}",
+  "mediaDimensions": "Dimensions: {{ width }} x {{ height }}",
```

Comment:
This is a different `𝗑` than the original one. I think the one originally used is more associated with multiplication rather than being a letter.